chpl/chpl-resources/src/main/resources/environment.properties.template

```diff
@@ -18,6 +18,14 @@ surveillanceThresholdToProcessAsJob=50
 #file name that daily SED details will be written to
 SEDDownloadName=chpl-sed-all-details.csv
 
+# number of tokens per API Key per time unit below
+rateTokenLimit=5
+# time unit where tokens above will be used
+# values are: second, minute, hour
+rateLimitTimeUnit=minute
+## api keys to be whitelisted seperated by commas
+whitelist=12909a978483dfb8ecd0596c98ae9094
```

Comment:
Should not have "real" API key committed to github. Should be something like:
`whitelist=api_key_here,second_api_key`